.azurePipeline/getPodsLogs.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+"""
+The script will currently be used in the Azure Pipeline to get back the logs of the pods when a test is failing
+allowing developers to document the issue.
+"""
+
+import json
+from pathlib import Path
+import subprocess
+import sys
+
+# The namespace is not configurable as we are expected to run this script only in Azure Dev-Ops which deployed the
+# service only in this namespace.
+NAMESPACE = "test-azure"
+
+
+def get_list_pods(_release_name):
+    cmd = "kubectl --namespace {} get pods -l release={} -o json".format(NAMESPACE, _release_name)
```

Comment:
Did you consider installing the python-kubernetes package and using that instead of shelling out?

https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/CoreV1Api.md#read_namespaced_pod_log

Reply:
I didn't even look if there was a package as I'm really used to the shell commands.
But I'll open an issue to record it (#446 ). To be honest, I also like the fact that the script does not have any requirement.